### ISSUE TITLE
Merging request rejected

### ISSUE THREAD

**Issue by jirkakocian** [edgeryders/annotator_store-gem#190]
Hi, I have been merging my codes and since yestreday evening the system is returninng the following error. Could you check it out pleasse? (creator_id=4779)

The change you wanted was rejected.
Maybe you tried to change something you didn't have access to.

**Comment by damingo** [comment]
Hi, to my knowledge this error only appears when a parent code (A) is merged into a child code of the parent (A → B). Merging A into B isn't possible, as in this case all child codes of A need to be moved below B - but the child codes of A include B itself. Ideally this needs to be prevented by the UI.
If this isn't the reason for the error you are experiencing, please let me know the codes you want to merge.

**Comment by jirkakocian** [comment]
Thanks for the prompt reaction, I assumed I might have made a mistake of such kind but actually it appeared when merging directly unrelated codes existing within the same hierarchy in parallel on the same level e.g.:
ID 4115	health → medical treatment → misdiagnosis
ID 4121	health → medical treatment → contradictory diagnosis-treatments (merging into above)
I am quite sure I was performign analogical operations and yesterday evening until at one point it just stopped working:) It seems that merging horizontally across hierarchies in other cases generates the same kind of problem

**Comment by damingo** [comment]
There were some records with inconsistent data which prevented codes from being merged. Probably due to the import of codes performed a few weeks ago. This is now fixed and merging codes works again.
